index.d.ts

```diff
@@ -79,6 +84,7 @@ declare namespace contextMenu {
 		readonly saveImage: (options: ActionOptions) => MenuItem;
 		readonly saveImageAs: (options: ActionOptions) => MenuItem;
 		readonly copyImageAddress: (options: ActionOptions) => MenuItem;
+		readonly copyImage: (options: ActionOptions) => MenuItem;
```

Comment:
Not the correct order. The order here should follow the order shown in the menu.